tests/test_pytest.py

```diff
@@ -37,7 +39,7 @@ def test_flexmock_teardown_works_with_pytest_part2():
     mock.method2()
 
 
-class TestForPytest(test_flexmock.RegularClass):
+class TestForPytest(FlexmockTestCase):  # pylint: disable=too-many-ancestors
```

Comment:
How about we just disable this error?

Reply:
Done! https://github.com/flexmock/flexmock/pull/114/commits/f178f5c5286831f84731b001c9fe340553852c79